src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -2126,7 +2126,7 @@ export function registerTerminalActions() {
 		constructor() {
 			super({
 				id: TerminalCommandId.OpenWebLink,
-				title: { value: localize('workbench.action.terminal.openLastUrlLink', "Open Last Url Link"), original: 'Open Last Url Link' },
+				title: { value: localize('workbench.action.terminal.openLastUrlLink', "Open Last URL"), original: 'Open Last URL' },
```

Comment:
Despite being a pleonasm, we want it to be easily discoverable in the command palette by searching for "link". I read it more like Open Last (URL) Link. The caps are a good idea though

Reply:
Cool. I've amended this PR to "Open Last URL Link".